### PR TITLE
Deploy script recommendations

### PR DESCRIPTION
**Pull request by VladimirFilonov**

### CODE REVIEW

File: fabfile/db/__init__.py [hunk 1]
```diff
@@ -102,9 +103,27 @@ def run(self, action, suffix=None):
         except KeyError:
             print('Error: Unknown engine `%s`' % self.engine)
 
+    # VF: @staticmethod
```
Comment (maxsocl) [inline fabfile/db/__init__.py]: We can't use @staticmethod for decorator (postgres fund is a decorator). I can move this func outside the BaseTask class but for consistency I prefered to leave this func inside the class.

Comment (sengottuvelk) [inline fabfile/db/__init__.py]: We can remove the entire line to keep the code clean.

**Comment by sengottuvelk** [comment]
@VladimirFilonov 
we can change the `# VF:` comments to `XXX or TODO or FIXME`, where we can indentity the code blocks across the code base easily which needs these improvements in future .

**Comment by sengottuvelk** [comment]
Otherwise Looks good to me. Merge when ready.

**Pull request closed without merging**